katz/src/test/kotlin/laws/FunctorLaws.kt

```diff
@@ -0,0 +1,32 @@
+package katz
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object FunctorLaws {
+
+    inline fun <reified F> laws(AP: Applicative<F> = applicative<F>()): List<Law> =
```

Comment:
why use an Applicative here, if we are testing Functor laws here? should it be convenient use Functor<F> instead of Applicative<F> ins't it?